sopel_modules/imdb/imdb.py

```diff
@@ -122,7 +122,7 @@ def run_omdb_query(params, verify_ssl, api_key, add_url=True):
     return message
 
 
-@url(r'https?:\/\/(?:www\.)?imdb\.com\/title\/(tt[0-9]+)')
+@url(r'https?:\/\/(?:(?:m|www)\.)?imdb\.com\/title\/(tt[0-9]+)')
```

Comment:
I don't think you need the `?:` before the m, since it's included right before it

Reply:
If I take that out, then `match.group(1)` will be either `m` or `www`. "Non-capturing" status does not apply recursively.

Reply:
What _would_ work is this, but the character savings are minimal.

```suggestion
@url(r'https?:\/\/(?:m\.|www\.)?imdb\.com\/title\/(tt[0-9]+)')
```

Reply:
i'm not necessarily looking for arbitrary character saving, I was just confused by the nested :?, would (?:m\.|?:www\.) work?

Reply:
Sounds like you wanted what I suggested in the second comment. (Which of course I thought about 5 seconds _after_ submitting the first.) If that suggestion works, I'll apply it later. About to start a meeting.

Reply:
Sounds good!

Reply:
Amended.